src/test/java/woohakdong/server/api/service/inquiry/InquiryServiceTest.java

```diff
@@ -0,0 +1,82 @@
+package woohakdong.server.api.service.inquiry;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.springframework.beans.factory.annotation.Autowired;
+import woohakdong.server.api.controller.inquiry.dto.InquiryRequest;
+import woohakdong.server.api.controller.inquiry.dto.InquiryResponse;
+import woohakdong.server.api.service.SecurityContextSetUp;
+import woohakdong.server.domain.inquiry.Inquiry;
+import woohakdong.server.domain.inquiry.InquiryCategory;
+import woohakdong.server.domain.inquiry.InquiryRepository;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import woohakdong.server.domain.member.Member;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.tuple;
+
+
+class InquiryServiceTest extends SecurityContextSetUp {
+
+    @Autowired
+    private InquiryService inquiryService;
+
+    @Autowired
+    private InquiryRepository inquiryRepository;
+
+    private Member member;
+
+    @BeforeEach
+    void setUp() {
+        member = createExampleMember();
+    }
+
+    @DisplayName("문의를 생성할 수 있다.")
+    @Test
+    void createInquiry() {
+        // Given
+        InquiryRequest request = new InquiryRequest("Test inquiry content", InquiryCategory.INQUIRY);
+
+        // When
+        inquiryService.createInquiry(request);
+
+        // Then
+        List<Inquiry> inquiries = inquiryRepository.getAll();
+        assertThat(inquiries).hasSize(1);
```

Comment:
```suggestion
        assertThat(inquiries).hasSize(1)
                .extracting(Inquiry::getInquiryContent, Inquiry::getInquiryCategory)
                .containsExactly(tuple("Test inquiry content", InquiryCategory.INQUIRY));
```

이거 이렇게 메소드 체이닝이 가능합니다! 가독성 부분에서 이렇게 수정하는 것이 좋을 것 같아요

Reply:
네네 수정하겠습니다.